### PR TITLE
Add initial docs for the napari plugin and Python api

### LINKED ISSUE #60: Add documentation for `epitools` package

### PR DESCRIPTION
**Pull request by p-j-smith**
Fixes #60 

- using sphinx to build the docs; hosted on readthedocs at http://epitools.readthedocs.io/ (we'll need to merge into `main` before the docs are built)

- updated the docstrings of the public functions  

- using the README as the landing page for the documentation

- using the [pydata sphinx theme](https://pydata-sphinx-theme.readthedocs.io/en/stable/index.html) with dark-mode disabled as the epitools logo only supports light-mode

- I've recorded several short movies showing how to use the plugin (they're all in `docs/_static/`). It should be possible to add them in `docs/napari_plugin/index.rst` using the [sphinx-contrib-video](https://pypi.org/project/sphinxcontrib-video/) extension, but the movies might first need to be converted form `.mov` to `.mp4`

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+# -- Project information -----------------------------------------------------
+
+project = "EpiTools"
+year = datetime.datetime.now(tz=pytz.timezone("GMT")).year
+author = "EpiTools Authors"
+copyright = f"2023-{year}, {author}"  # noqa: A001
+
+# The full version, including alpha/beta/rc tags
+release = f"v{epitools.__version__}"
```
Comment (paddyroddy) [inline docs/conf.py:31]: Have sent you a message about this

Comment (paddyroddy) [inline docs/conf.py:31]:
```suggestion
version = f"v{epitools.__version__}"
```

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+version = "dev" if "dev" in release else release
```
Comment (paddyroddy) [inline docs/conf.py:32]: Will be handled by `setuptools-scm`
```suggestion
release = version
```

**Review by paddyroddy — APPROVED** [review]
Pending changes from my comments looks great. Nice work doing this so quickly! 👶 🍼 🐤 🚼 👨‍🍼

File: docs/conf.py (hunk 1 above)
Comment (p-j-smith) [reply docs/conf.py:31]: thanks! it's fixed

**Comment by p-j-smith** [comment]
Thanks for the speedy review! Time for me to👨‍🍼

**Comment by paddyroddy** [comment]
Getting `/Users/paddy/arc/napari/epitools/epitools/docs/api/index.rst: WARNING: document isn't included in any toctree`

**Comment by paddyroddy** [comment]
I'll merge and fix anything I find later!

**Pull request merged**